src/enclave-agent/src/config.rs

```diff
@@ -0,0 +1,119 @@
+use serde::{Deserialize, Serialize};
+use std::{fs::File, io::Write, path::Path};
+
+const DEFAULT_KEYPROVIDER_CONFIG_PATH: &str = "/etc/ocicrypt_keyprovider_native.conf";
+const DEFAULT_KEYPROVIDER_CONFIG: &str = r#"
+{
+    "key-providers": {
+        "attestation-agent": {
+            "native": "attestation-agent"
+        }
+    }
+}
+"#;
+
+#[derive(Serialize, Deserialize, Default)]
+pub struct DecryptConfig {
+    pub key_provider: Option<String>,
+    #[serde(default = "default_security_validate")]
+    pub security_validate: Option<bool>,
```

Comment:
Another flag is the switch for `auth`. related link for `ImageConfig` https://github.com/confidential-containers/image-rs/blob/main/src/config.rs#L29. I'm afraid we also need to add one flag for that?

Reply:
The flag for `auth` has been added.

Reply:
Great, sorry for neglecting